routes/home.js

```diff
@@ -48,7 +48,13 @@ routes.joinGame = function(req, res) {
       console.error("Couldn't find the specified game! ", err);
       res.status(500).send("Couldn't find the specified game");
     } else {
-    	res.send(game);
+    	User.find({_id: game.host}, function(err, host) {
+        res.send({
+          game: game,
```

Comment:
Then send only the game object with populated host field. 
(Maybe send currUser)